spec/test-suite.json

```diff
@@ -0,0 +1,71 @@
+{
+    "suite":
+    {
+        "zmanim":
+        [
+            {
+                "setup":
+                {
+                    "calculator":
+                    {
+                        "name": "SunTimes",
+                        "zmanimInformation":
+                        {
+                            "name": "",
+                            "url": ""
+                        },
+                        "information":
+                        {
+                            "name": "US Naval Observatory's Almanac for Computer algorithm",
+                            "url": "http://aa.usno.navy.mil/publications/docs/asa.php"
+                        }
+                    },
+                    "date": "2010-04-02",
+                    "location":
+                    {
+                        "name": "Lakewood, NJ",
+                        "lat": 40.09596,
+                        "long": -74.22213,
+                        "elevation": 0,
+                        "timeZone":
+                        {
+                            "offset": -5,
+                            "zoneInfoName": "America/New_York"
+                        }
+                    }
+                },
+                "tests":
+                [
+                    {
+                        "method": "sunset",
+                        "description": "Gets the exact sunset time",
+                        "extra":
+                        {
+                            "edgeCase": false,
+                            "information": ""
+                        },
+                        "expected":
+                        {
+                            "utc": "2010-04-02T12:22:3:710Z",
```

Comment:
ISO8601 should render 0-padded seconds (and other time increments).  I'm guessing you mean to use this standard because of the 'T' separator, please clarify if otherwise.

Reply:
Yes this would be in a valid ISO8601 format this is probably just a typo.